workorders.lic

```diff
@@ -632,6 +632,7 @@ class WorkOrders
       else
 		volume = deed_ingot_volume
         fput('tap deed')
+        volume = deed_ingot_volume
```

Comment:
This doesn't look right, I think it's a duplicate of line 633.

Reply:
Fixed.